_posts/2019-08-12-iot-security-Issues-and-challenges.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+current: post
+cover: iotsecIssues-blog.png
+title: IoT Security Issues and Challenges
+date: 2019-08-12 13:00:00
+category: [news,thoughts]
+author: Nikoletta Triantafyllopoulou
+tags: [Open Source]
+class: post-template
+---
+{% asset blog/IoTSecIssues-Blog.png srcset:width="1300 2x" srcset:width="650 1x" alt="IoTSecurityIssues" %}
+
+*IoT Security Issues and Challenges* 
+IoT Security issues are quite a challenge for this burgeoning industry. There is a growing number of devices, applications and systems which turn to IoT. This helps them to upscale their efficiency and their services and to make life easier for users. So let us take a closer look at the industry.
+
+## *The Internet of Things* ##
+This buzzing, new industry sports cutting-edge technology. It enhances digitisation while at the same time it produces demanding challenges.
+But before we delve into these in more detail, let us have a look at the state of IoT at the moment.  
+
+According to a report by [IoT Analytics] (https://iot-analytics.com/state-of-the-iot-update-q1-q2-2018-number-of-iot-devices-now-7b/) there are over 17 Billion devices connected to IoT. As one can understand, this number will only multiply in the coming years. The devices connected to IoT range greatly.  From Smart Home Automation devices, routers, printers to thermostats, refrigerators and cameras. Amazon Alexa and Google Assistant too. Smart locks, smart watches, ATMs, industrial machines, manufacturing processes and applications. Smart clothing items, drones, solar panels, automobile navigation, infotainment systems, advanced medical devices.  The list is long and it is updated as more types of devices, gadgets and applications are added.
+
+Regardless of the technology or the number  of IoT devices, the main challenge remains one: That is *Security* and everything about it.
+
+##IoT Security Issues##
+### * *Unsecured Devices*
```

Comment:
Why are we using bold *inside* a header?

Reply:
I thought that the author of the post so wants, so in addition to image processing, I did not make changes to the text

Reply:
I see - OK revert those changes as we want the look and feel to be universal.

Reply:
last commit corrects headings and subheadings